mailbox/api/src/test/java/org/apache/james/mailbox/model/AttachmentTest.java

```diff
@@ -31,43 +32,45 @@
 
 public class AttachmentTest {
 
+    private static Charset CHARSET = Charset.defaultCharset();
```

Comment:
Done.
@rouazana I understand the idea. But in really no matter the CHARSET, string ° bytes = bytes ° string = identity. I don't think even if Charset change accross computer this will change anything about our tests.